compass/ocean/inactive_top_cells.py

```diff
@@ -0,0 +1,28 @@
+import os
+import xarray
+
+from mpas_tools.io import write_netcdf
+
+
+def remove_inactive_top_cells_output(in_filename, inactive_top_cells=1):
+    """
+    Remove inactive top cells from the output netCDF file
+
+    Parameters
+    ----------
+    in_filename : str
+        Filename for the netCDF file to be cropped
+
+    inactive_top_cells : str, optional
+        The number of inactive top cell layers. It should be equal to
+        ``config.inactive_top_cells``.
+    """
+    if not os.path.exists(in_filename):
+        raise OSError('File {} does not exist.'.format(in_filename))
+
+    out_filename = in_filename.split('.')[0] + '_crop.nc'
+
+    with xarray.open_dataset(in_filename) as ds_in:
+        ds_out = ds_in.isel(nVertLevels=slice(1, None))
```

Comment:
Here would be my suggested way of handling the offset.  The `inactive_top_cells` would no longer be needed, which is important for implementation elsewhere.

```suggestion
        minLevelCell = ds.minLevelCell
        minval = minLevelCell.min().values
        maxval = minLevelCell.max().values
        if minval != maxval:
            raise ValueError('Expected minLevelCell to have a constant value '
                             'for inactive top cell tests')
        ds_out = ds_in.isel(nVertLevels=slice(minval, None))
```

Reply:
Hmm, there's a complication with my suggestion.  The mesh isn't included in `output.nc` files you default.  We could add it but I think the better solution is also supplying a mesh file name (`None` by default) so you can get `minLevelCell` from it.